Sprint-2/debug/0.js

```diff
@@ -1,7 +1,9 @@
 // Predict and explain first...
+//a, b are not number, returns undefined when using console.log inside function body
```

Comment:
Can you explain a bit more what you mean by "a, b are not number" ?

Reply:
I mean (a, b) are parameters, they could be anything when you want to pass a value to run thw function. number or string.